tests/e2e_logging.py

```diff
@@ -324,38 +324,70 @@ def require_new_response(r):
 @reqs.supports_methods("custom_auth")
 def test_custom_auth(network, args):
     if args.package == "liblogging":
-        primary, _ = network.find_primary()
+        primary, other = network.find_primary_and_any_backup()
 
-        with primary.client("user0") as c:
-            LOG.info("Request without custom headers is refused")
-            r = c.get("/app/custom_auth")
-            assert r.status_code == http.HTTPStatus.UNAUTHORIZED.value, r.status_code
-
-            name_header = "x-custom-auth-name"
-            age_header = "x-custom-auth-age"
-
-            LOG.info("Requests with partial headers are refused")
-            r = c.get("/app/custom_auth", headers={name_header: "Bob"})
-            assert r.status_code == http.HTTPStatus.UNAUTHORIZED.value, r.status_code
-            r = c.get("/app/custom_auth", headers={age_header: "42"})
-            assert r.status_code == http.HTTPStatus.UNAUTHORIZED.value, r.status_code
-
-            LOG.info("Requests with unacceptable header contents are refused")
-            r = c.get("/app/custom_auth", headers={name_header: "", age_header: "42"})
-            assert r.status_code == http.HTTPStatus.UNAUTHORIZED.value, r.status_code
-            r = c.get(
-                "/app/custom_auth", headers={name_header: "Bob", age_header: "12"}
-            )
-            assert r.status_code == http.HTTPStatus.UNAUTHORIZED.value, r.status_code
+        for node in (primary, other):
```

Comment:
The below is just whitespace+formatting adjustment, because of the additional loop here.